src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -562,9 +646,17 @@ private Conversion ClassifyStandardImplicitConversion(BoundExpression sourceExpr
         }
 
         private Conversion ClassifyStandardImplicitConversion(TypeSymbol source, TypeSymbol destination, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
+        {
+            var conversion = ClassifyStandardImplicitConversionInternal(source, destination, ref useSiteInfo);
+            Debug.Assert(conversion.Kind == ConversionKind.NoConversion || IsStandardImplicitConversionFromType(conversion.Kind));
+            return conversion;
+        }
+
+        private Conversion ClassifyStandardImplicitConversionInternal(TypeSymbol source, TypeSymbol destination, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
```

Comment:
>ClassifyStandardImplicitConversionInternal [](http://example.com/codeflow?start=27&length=42)

It looks like there is only one call site. Consider making this a local function.